src/components/ChartTool/Charts.js

```diff
@@ -1,67 +1,68 @@
-import React from 'react'
+import React, { useState, useEffect, useRef } from 'react'
 import styles from './Charts.module.css'
 import Chart from 'chart.js'
 
-class Charts extends React.Component {
-    state = {
-        
-    }
+const Charts = ({ stats, bases, name, color }) => {
+  const [statsName] = useState(stats)
+  const [basesData] = useState(bases)
+  const [pokemonName] = useState(name)
+  const [pokemonColor] = useState(color === 'white' ? 'black' : color )
```

Comment:
for the previous constants, if there is no "setState" that means it doesn't change, if it doesn't change it should not be part of the state but a prop, therefore you shouldn't use the useState hook here